tripal_chado/src/TripalStorage/ChadoRecords.php

```diff
@@ -154,14 +161,14 @@ protected function initTable($elements) : bool {
         'tables' => [],
         'record_id' => 0,
       ];
-      // The base table doesn't have an alias so make sure we have an entry for it..
+      // The base table doesn't have an alias so make sure we have an entry.
       $this->records[$base_table]['tables'][$base_table] = [
         'chado_table' => $base_table,
         'items' => [],
       ];
     }
 
-    // if the table has not been initialized then do so.  The tables
+    // If the table has not been initialized then do so.  The tables
```

Comment:
```suggestion
    // If the table has not been initialized then do so. The tables
```
In case you care about double spaces...